src/ansiblelint/rules/name.py

```diff
@@ -35,14 +60,23 @@ def matchtask(
                 tag="name[missing]",
                 filename=file,
             )
+        return (
+            self._check_name(name, filename=file, linenumber=task[LINE_NUMBER_KEY])
+            or False
+        )
+
+    # pylint: disable=useless-return
+    def _check_name(
+        self, name: str, filename: "Optional[Lintable]", linenumber: int
```

Comment:
```suggestion
        self, name: str, filename: Optional[Lintable], linenumber: int
```

If that is a Lintable, should the var be named `lintable` instead of `filename`?